<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MobileDialog</class>
 <widget class="QDialog" name="MobileDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mobile-ID</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
color: #07142A;
font-family: Roboto, Helvetica;
font-size: 14px;
}
#MobileDialog {
background-color: #FFFFFF;
border-radius: 4px;
}
#label {
color: #003168;
font-size: 20px;
font-weight: 700;
}
#errorCode, #errorPhone {
color: #AD2A45;
}
QLineEdit {
padding: 10px 14px;
border: 1px solid #C4CBD8;
border-radius: 4px;
background-color: white;
placeholder-text-color: #607496;
font-size: 16px;
}
QCheckBox {
spacing: 8px;
border: none; /*Workaround for right padding*/
}
QCheckBox:disabled {
color: #C4CBD8;
}
QCheckBox::indicator {
width: 16px;
height: 16px;
}
QCheckBox::indicator:unchecked {
image: url(:/images/icon_checkbox.svg);
}
QCheckBox::indicator:unchecked:hover {
image: url(:/images/icon_checkbox_active.svg);
}
QCheckBox::indicator:unchecked:disabled {
image: url(:/images/icon_checkbox_disabled.svg);
}
QCheckBox::indicator:checked {
image: url(:/images/icon_checkbox_check.svg);
}
QCheckBox::indicator:checked:hover {
image: url(:/images/icon_checkbox_check_active.svg);
}
QCheckBox::indicator:checked:disabled {
image: url(:/images/icon_checkbox_check_disabled.svg);
}
QPushButton {
padding: 12px 12px;
border-radius: 4px;
border: 1px solid #AD2A45;
color: #AD2A45;
font-weight: 700;
}
QPushButton:hover {
background-color: #F5EBED;
}
QPushButton:pressed {
background-color: #E1C1C6;
}
QPushButton:default {
color: #ffffff;
border-color: #2F70B6;
background-color: #2F70B6;
}
QPushButton:default:hover {
border-color: #2B66A6;
background-color: #2B66A6;
}
QPushButton:default:pressed {
border-color: #215081;
background-color: #215081;
}
QPushButton:default:disabled {
border-color: #82A9D3;
background-color: #82A9D3;
}</string>
  </property>
  <layout class="QVBoxLayout" name="MobileDialogLayout">
   <property name="spacing">
    <number>40</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Enter your phone number to sign with mobile-ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutContent">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="layoutPhone">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPhone">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Country code and phone number</string>
         </property>
         <property name="buddy">
          <cstring>phoneNo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="phoneNo">
         <property name="placeholderText">
          <string notr="true">37254321</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorPhone">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutCode">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Personal code</string>
         </property>
         <property name="buddy">
          <cstring>idCode</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="idCode">
         <property name="placeholderText">
          <string notr="true">47101010033</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorCode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRemember">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Remember me</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sign">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Sign</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
